appcues/src/main/java/com/appcues/data/mapper/trait/TraitsMapper.kt

```diff
@@ -15,6 +16,9 @@ internal class TraitsMapper(
                     add(factory.invoke(it.first.config, it.second))
                 }
             }
+
+            // ensure BackdropKeyholeTrait is applied first
+            sortBy { it is BackdropKeyholeTrait }
```

Comment:
@iujames what do you think? its the easiest way to go about it until we actually need to change our public API

Reply:
yeah, I'd be in favor of keeping this an internal special case to start, until we discover we need something more general.